docs/dev/agent_dev_env.md

```diff
@@ -27,6 +27,40 @@ You must install [go](https://golang.org/doc/install) version 1.9.2 or above. Ma
 sure that `$GOPATH/bin` is in your `$PATH` otherwise Invoke cannot use any
 additional tool it might need.
 
+### Python
```

Comment:
this `Python` section mostly talks about installing the dev versions of python needed to build the agent against. So I think it should be left below in the `System or Embedded` section.

Here, you could simply mention in the `Invoke` section that a standard python+pip install is required for development maybe? What do you think?